src/Propel/Runtime/ActiveQuery/Criteria.php

```diff
@@ -1247,6 +1255,41 @@ public function hasSelectModifier($modifier)
         return in_array($modifier, $this->selectModifiers);
     }
 
+    /**
+     * @return null|Lock Get lock value.
```

Comment:
Can we use FQCN in docblocks here, please?

Reply:
sure